docs/uglify-examples.md

```diff
@@ -238,6 +238,41 @@ grunt.initConfig({
 });
 ```
 
+
+## Compiling all files separately on the each their path
+
+This configuration will compress and mangle all js files separately on the each their path.
+
+Also except jQuery for mangling and ignore all `*.min.js` files.
+
+```js
+// Project configuration.
+uglify: {
+	dev: {
+		options: {
+			mangle: {
+				except: ['jQuery']
+			}
+		},
+		files: [{
+			expand: true,
+			src: ['dist/assets/js/*.js', '!dist/assets/js/*.min.js'],
+			dest: 'dist/assets',
+			cwd: '.',
+			rename: function (dst, src) {
+				// To keep src js files and make new files as *.min.js :
+				// return dst + '/' + src.replace('.js', '.min.js');
+				// Or to override to src :
+				return src;
+			}
+		}]
+	}
+},
+
+```
+
```

Comment:
No need for 3 line breaks use one and remove the one before the closing three backticks.